net-p2p/amule/amule-2.3.1-r1.ebuild

```diff
@@ -0,0 +1,108 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit eutils wxwidgets user
+
+MY_P=${PN/m/M}-${PV}
+S="${WORKDIR}"/${MY_P}
+
+DESCRIPTION="aMule, the all-platform eMule p2p client"
+HOMEPAGE="http://www.amule.org/"
+SRC_URI="mirror://sourceforge/${PN}/${MY_P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="daemon debug geoip nls remote stats unicode upnp +X"
+
+DEPEND=">=dev-libs/crypto++-5
+	>=sys-libs/zlib-1.2.1
+	sys-libs/binutils-libs:0=
+	stats? ( >=media-libs/gd-2.0.26:=[jpeg] )
+	geoip? ( dev-libs/geoip )
+	upnp? ( >=net-libs/libupnp-1.6.6 )
+	remote? ( >=media-libs/libpng-1.2.0:0=
+	unicode? ( >=media-libs/gd-2.0.26:= ) )
+	X? ( >=x11-libs/wxGTK-2.8.12:2.8[X] )
+	!X? ( >=x11-libs/wxGTK-2.8.12:2.8 )
+	!net-p2p/imule"
+RDEPEND="${DEPEND}"
+
+pkg_setup() {
+	if use stats && ! use X; then
+		einfo "Note: You would need both the X and stats USE flags"
+		einfo "to compile aMule Statistics GUI."
+		einfo "I will now compile console versions only."
+	fi
+}
+
+pkg_preinst() {
+	if use daemon || use remote; then
+		enewgroup p2p
+		enewuser p2p -1 -1 /home/p2p p2p
```

Comment:
Yes, renaming the user sounds good. However, I'd say this either needs to go through net-p2p or you need to become proxied maintainer.